src/main.rs

```diff
@@ -257,6 +304,14 @@ fn main() {
     }
     data.insert("tooltip", tooltip);
 
+    let css_class = current_condition[lang.weather_desc()][0]["value"]
+        .as_str()
+        .unwrap()
+        .to_string()
+        .to_lowercase();
+
+    data.insert("class", css_class);
```

Comment:
this looks cool - how are you using it?

Reply:
I was using it to change the background color of the widget depending on the condition